client/admin_test.go

```diff
@@ -123,7 +123,12 @@ func Example_accounting() {
 // Example_permission shows how to use the admin client to
 // get/set/list/delete permission configs.
 func Example_permission() {
-	s := server.StartTestServer(nil)
+	// Don't use the high-level function to start the test server,
```

Comment:
Could we just adjust this test instead to tolerate the existence of `test-user`? Or delete the user after the test has started instead of bypassing its creation?